TransformationSystem/scripts/dirac-transformation-replication.py

```diff
@@ -0,0 +1,57 @@
+#!/bin/env python
+"""
+Create a production to replicate files from some storage elments to others
+
+Example::
+
+  dirac-ilc-replication-transformation <MetaValue> <TargetSEs> -S<SourceSEs> -G<GroupSize> -NExtraName
+
+Options:
+   -G, --GroupSize <value>     Number of Files per transformation task
+   -S, --SourceSEs <value>     SourceSE(s) to use
+   -N, --Extraname <value>    String to append to transformation name in case one already exists with that name
+   -T, --TransformationType <value>  Which transformation type to use, default 'Replication'
+   -K, --MetaKey <value>       Which MetaKey to use
+   -M, --MetaData <value>      Key-Value Pair to use in addition to the default MetaKey and Value
+
+
+:since:  May 31, 2018
+:author: A. Sailer
+"""
+from DIRAC.Core.Base import Script
+from DIRAC import gLogger, exit as dexit
+
+__RCSID__ = "$Id$"
+
+
+def _createTrafo():
+  """reads command line parameters, makes check and creates replication transformation"""
+  from DIRAC.TransformationSystem.Utilities.ReplicationCLIParameters import Params
+  clip = Params()
+  clip.registerSwitches(Script)
+  Script.parseCommandLine()
+  if not clip.checkSettings(Script)['OK']:
+    gLogger.error("ERROR: Missing settings")
+    return 1
+  from DIRAC.TransformationSystem.Utilities.ReplicationTransformation import createDataTransformation
```

Comment:
This may just be my taste, but I prefer imports grouped at the beginning, if possible.

Reply:
I don't think this can be imported before the parseCommandLine()

Reply:
At the moment, It seems things are still working when moving the imports to the top.